docs/can-slot.md

```diff
@@ -0,0 +1,35 @@
+@typedef {can-stache.sectionRenderer} can-component/can-slot <can-slot>
+@parent can-component.elements
+
+@description Positions `<can-template />` elements with matching name attribute using current scope.
+
+@signature `<can-slot name='templateName' />`
```

Comment:
The params aren't detailed.  We'd normally CAPITALIZE `templateName` and specify that as a param.